perfscale_regression_ci/scripts/scalability/replace_loadedworker_nodes.sh

```diff
@@ -0,0 +1,73 @@
+#/!/bin/bash
+##################################################################################################
+## Auth=svetsa@redhat.com
+## Desription: Script for resizing worker nodes with larger worker nodes
+## Polarion test case: OCP-40557 - Replace loaded worker nodes with larger instances
+## https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-40557
+## Customer related: https://access.redhat.com/support/cases/#/case/02864921
+## Pre-requisites: 
+### 1) Prior to running this script, make sure to run loaded-upgrade Jenkins job 
+### to create a new cluster with one of the below profiles and run cluster-density workload
+### https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/loaded-upgrade/
+### Cluster with 3 master nodes and 30 worker nodes using one of these profiles:
```

Comment:
Cluster with 3 m5.2xlarge master nodes and 30 m5.xlarge worker cluster on AWS, or equivalent size of these profiles: